hoot-core/src/main/cpp/hoot/core/io/HootNetworkRequest.h

```diff
@@ -42,11 +42,11 @@ namespace hoot
 {
 
 /** Class for sending HTTP network requests to an OSM API */
-class OsmApiNetworkRequest
+class HootNetworkRequest
```

Comment:
Renamed the `OsmApiNetworkRequest` class to a more generic name `HootNetworkRequest` because it is just a network request that can be made to WFS or OSM API or any REST endpoint.